### PR TITLE
Add wait timeout for allocated container registration

### PR DESCRIPTION
**Pull request by morenn520**
https://github.com/linkedin/TonY/issues/411

### CODE REVIEW

**Comment by oliverhu** [comment]
Thanks for the PR! Could you make REGISTRATION_TIMEOUT_MS a configurable variable? 5 minutes is too short, I'd use 15 minutes as the default time out interval.

**Comment by morenn520** [comment]
@oliverhu I have make it as configurable variable. Review please.

File: tony-core/src/main/resources/tony-default.xml [hunk 1]
```diff
@@ -145,6 +145,13 @@
     <value>1</value>
   </property>
 
+  <!-- Container configurations -->
+  <property>
+    <description>Timeout, in milliseconds for allocated container registration before forcibly killing application.</description>
+    <name>tony.container.timeout</name>
```
Comment (oliverhu) [inline tony-core/src/main/resources/tony-default.xml]: rename to "tony.container.allocation.timeout"

Comment (morenn520) [reply tony-core/src/main/resources/tony-default.xml]: Code updated. @oliverhu Please review again. Thanks.

**Pull request merged**